src/app/components/harvest/pages/list/list.component.html

```diff
@@ -42,6 +42,28 @@ <h1>Upload History</h1>
       <a class="btn btn-sm btn-primary" [bawUrl]="row.viewUrl">
         {{ asHarvest(row).status !== "complete" ? "Continue" : "View" }}
       </a>
+      <a
+        id="list-abort-button"
```

Comment:
should not be using a an id for a repeated element

Reply:
*maybe* try 
```suggestion
        name="abort-button"
```

instead